File: Data Structures & Algorithms/maximum-product-subarray/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    maxProduct(nums) {
        let max = nums[0];
        for (let i = 0; i < nums.length; i++) {
            let sum = 1;
            for (let j = i; j < nums.length; j++) {
                sum *= nums[j];
                max = Math.max(sum, max)
            }
        }

        return max;
       
    }
}

File: Data Structures & Algorithms/single-number/submission-0.py
class Solution:
    def singleNumber(self, nums: List[int]) -> int:
        count = {}
        for n in nums:
            count[n] = count.get(n)+ 1 if count.get(n) else 1
        
        for key, val in count.items():
            if val == 1:
                return key
        return -1
        